File: questão2.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int nota1 = 0;
    int nota2 = 0;
    float media = 0;
    int notaEx = 0;
    float media2 = 0;

    printf("Digite a primeira nota: ");
    scanf("%d", &nota1);

    printf("Digite a segunda nota: ");
    scanf("%d", &nota2);

    media = (nota1 + nota2)/2;

    if(media>=70){
        printf("Aprovado!");
    }
    else if(media>=40 && media<=69){
        printf("Digite a nota do exame: ");
        scanf("%d", &notaEx);

        media2 = (media + notaEx)/2;

        if(media2>=100){
            printf("Aprovado!")
        }else{
            printf("Reprovado!")
        }
        
    }
    else{
        printf("Reprovado!")
    }

}

File: questão2.py
nota1 = int(input("Digite a primeira nota: "))
nota2 = int(input("Digite a segunda nota: "))

media = (nota1 + nota2)/2

if(media>=70):
    print("Aprovado!")
elif(media>=40 and media<=69):
    notaEx = int(input("Digite a nota do exame: "))
    media2= (media + notaEx)/2
    if(media2>=100):
        print("Aprovado!")
    else:
        print("Reprovado!")
else:
    print("Reprovado!")

